contracts/truefi2/FixedTermLoanAgency.sol

```diff
@@ -226,6 +226,10 @@ contract FixedTermLoanAgency is IFixedTermLoanAgency, UpgradeableClaimable {
         emit BorrowingMutexChanged(newMutex);
     }
 
+    function setRateAdjuster(ITrueRateAdjuster newRateAdjuster) public onlyOwner {
+        rateAdjuster = newRateAdjuster;
+    }
```

Comment:
Can we avoid adding unnecessary setters to contracts?

Reply:
+1.

If we're launching the first version with TrueRateAdjuster (=> CreditModel) in the initializer, then we won't need to add a setter.